.env

```diff
@@ -21,3 +21,4 @@ SITE_NAME=''
 USER_INFO_COOKIE_NAME=''
 APP_ID=''
 MFE_CONFIG_API_URL=''
+LTI_INSTRUCTOR_EMBED=false
```

Comment:
Creating a new config setting that indicates whether to open the lti instructor tool in a new tab or whether to embed as an iframe. Since we are defaulting to opening in a new tab for now I'm just changing the env settings here, but let me know if this should be added to frontend-app-learning or somewhere else now.

Reply:
If used, this would need to be configurable on a per-tool basis so I don't think an env variable works here. It would apply the same value to all LTI tools.

We don't have anything on the data model to support configuring a value for embed vs. tab yet. Maybe just toss a constant in ExternalReviewDashboard.jsx and call it good for now?

Reply:
That makes sense! Sounds good.